### PR TITLE
Added remove_lab and remove_token commands

### PR DESCRIPTION
**Pull request by mgalka**
Added commands to remove tokens and labs from kernelci-backend.
The commands utilise /lab and /token delete handlers from the backend. 

remove_token command doesn't provide any extra checks apart from what's been implemented in the backend, therefore it's possible to e.g. remove a lab token without any notice.

### CODE REVIEW

**Review by khilman — APPROVED** [review]
Very nice, we've been missing these features.

File: kci_admin/lab.py [hunk 1]
```diff
@@ -78,3 +87,7 @@ def get_list(hostname):
         token_id = lab['token']['$oid']
         lab['token'] = tokens[token_id]['token']
     return data['result']
+
+
+def request_remove(hostname, id_):
+    return kci_request_delete(hostname, "/lab/{}".format(id_))
```
Comment (gctucker) [inline kci_admin/lab.py]: Nitpick: missing trailing newline

**Review by gctucker — CHANGES_REQUESTED** [review]
Commit messages should be in the present tense i.e. "add remove_token command" rather than "added".  Also the command is "remove_lab" but the commit message says "lab_remove".

I think the README.md needs to be updated slightly with these new commands.

Other than those cosmetic things, I've tested it OK with the backend fix applied from https://github.com/kernelci/kernelci-backend/pull/125, thanks!

File: kci_admin/lab.py (hunk 1 above)
Comment (mgalka) [reply kci_admin/lab.py]: Commit messages corrected
Trailing newline added
README.md updated

**Review by gctucker — APPROVED** [review]

**Comment by gctucker** [comment]
Thanks, this can now be merged.

See also the comments I put on https://github.com/kernelci/kernelci-backend/pull/125, which needs to be merged for the `delete_lab` command to work.

**Pull request merged**